src/components/FormElements/MuxedKeyPicker.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Input, InputProps } from "@stellar/design-system";
+
+interface MuxedKeyPickerProps extends Omit<InputProps, "fieldSize"> {
+  id: string;
+  fieldSize?: "sm" | "md" | "lg";
+  label: string;
+  labelSuffix?: string | React.ReactNode;
+  placeholder?: string;
+  readOnly?: boolean;
+  value: string;
+  error: string | undefined;
+  // eslint-disable-next-line no-unused-vars
+  onChange?: (e: React.ChangeEvent<any>) => void;
+}
+
+export const MuxedKeyPicker = ({
```

Comment:
:thinking: The same question here. It seems so similar to `PubKeyPicker` that we might use that instead.